modules/QnA/resources/qna.tmpl.html

```diff
@@ -50,7 +50,8 @@
                         <div data-bind='css:{  "qnaAnnouncementText" : !isRead(),
                                                 "qnaAnnouncementTextRead" : isRead()
                                             },
-                                        text: entries()[0]().getText()'>
+                                        html: entries()[0]().getText(),
+                                        click: $parent.openLinkInNewTab'>
```

Comment:
@MaksymKorniiov  
Does this resolve the UX of having pointer-cursor on the link?